.github/workflows/dispatch.yml

```diff
@@ -0,0 +1,32 @@
+name: Dispatch workflows
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  dispatch-dc:
+    name: Dispatch DC workflow
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v3    
+      - uses: dorny/paths-filter@v2
```

Comment:
do you have v3 for this

Reply:
There is only v2.
https://github.com/dorny/paths-filter